llvm/include/llvm/Support/ThreadPool.h

```diff
@@ -140,54 +142,74 @@ class ThreadPool {
             },
             std::move(F)};
   }
+};
+
+/// A ThreadPool implementation using std::threads.
+///
+/// The pool keeps a vector of threads alive, waiting on a condition variable
+/// for some work to become available.
+class StdThreadPool : public ThreadPoolInterface {
```

Comment:
Same here, `#if LLVM_ENABLE_THREADS` maybe? The definition in the .cpp has the define.

Reply:
Obsolete now that we test both I think, resolving.